scripts/importer/tests/data/reference.psql

```diff
@@ -1,4 +1,7 @@
-COPY beacon.beacon_dataset_counts_table (datasetid, dataset, callcount, variantcount) FROM stdin;
+COPY beacon.beacon_dataset_counts_table (datasetid, callcount, variantcount) FROM stdin;
+GRCh37p13:Dataset 1:2001-01-01 00:00:00	18	18
+GRCh37p13:Dataset 2:2001-01-01 00:00:00	12	15
```

Comment:
Running this gives me:
```
686c687
< COPY beacon.beacon_dataset_counts_table (datasetid, dataset, callcount, variantcount) FROM stdin;
---
> COPY beacon.beacon_dataset_counts_table (datasetid, callcount, variantcount) FROM stdin;
743,744c744,745
< GRCh13:Dataset 1:Version 1	\N	18	18
< GRCh13:Dataset 2:Version 1	\N	12	17
---
> GRCh13:Dataset 1:2001-01-01 00:00:00	18	18
> GRCh13:Dataset 2:2001-01-01 00:00:00	12	15
```

Reply:
Yes, looks like I somehow miscalculated the variantcount. Fixed in 28f42d1e065ba774ca4c0da0ea4085f4acdcb4db.